src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -49,7 +49,7 @@ class RsFileStub : PsiFileStubImpl<RsFile> {
     override fun getType() = Type
 
     object Type : IStubFileElementType<RsFileStub>(RsLanguage) {
-        private const val STUB_VERSION = 198
+        private const val STUB_VERSION = 199
```

Comment:
Some time ago we introduced two additional version fields: `org.rust.lang.core.parser.RustParserDefinition#LEXER_VERSION` and `org.rust.lang.core.parser.RustParserDefinition#PARSER_VERSION` for lexer and parser changes respectively. So in this case, you need to change `PARSER_VERSION`. Note, `PARSER_VERSION` depends on `LEXER_VERSION` and `STUB_VERSION` depends on `PARSER_VERSION`, so incrementing `PARSER_VERSION` also rebuilds stubs

Reply:
@Undin Thanks!